org.testeditor.web.backend.persistence/src/test/java/org/testeditor/web/backend/testexecution/TestExecutorIntegrationTest.xtend

```diff
@@ -48,18 +48,24 @@ class TestExecutorIntegrationTest extends AbstractPersistenceIntegrationTest {
 		val previousCommitID = '1234'
 		workspaceRoot.newFile(userId + '/' + TestExecutorProvider.LOG_FOLDER + '/testrun-SomeTest-200001011200123.yaml') => [
 			JGitTestUtil.write(it, '''
-				Source: "SomeTest"
-				CommitID: "«latestCommitID»"
-				Children:
-			'''.indent(1))
+				"started": "on some instant"
+				"resourcePaths": [ "one", "two" ]
```

Comment:
`TestSuiteResource::writeCallTreeYamlPrefix` actually writes "resources", not "resourcePaths"